### DISCUSSION TITLE
The font cannot be displayed properly, and some parts of the display are framed

### DISCUSSION THREAD

**Discussion by lanpeng1612** [freeplane/freeplane#1660] [Help]
1.因为使用的是精简win11系统,系统字体不全

2.软件默认使用SansSenif字体,无法正常显示

3.每创建一个节点都需要手动设置,因此很不方便

4.在网上找了一下相关解决方法,未能解决问题,软件内没有找到设置默认字体的设置

1. Because we are using a streamlined win11 system, the system fonts are incomplete.

2. The software uses SansSenif font by default and cannot be displayed normally.

3. Every time you create a node, you need to set it manually, so it is very inconvenient.

4. I searched for relevant solutions on the Internet, but failed to solve the problem. There was no setting for setting the default font in the software.

**Comment by euu2021** [comment]
> There was no setting for setting the default font in the software.

You can change the default font for the nodes using styles. Learn about styles at: https://docs.freeplane.org/user-documentation/styles.html?highlight=style#styles
